### PR TITLE
Connect to API 

### PR DESCRIPTION
**Pull request by peterdgreat**
- Refactored  add book and remove book features to persist your changes in the server.
- Refactored the redux code of my application using middleware to make async requests to the API.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @peterdgreat 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookLists.js [hunk 1]
```diff
@@ -15,9 +15,12 @@ const BookLists = (props) => {
...
   return (
 
-    <li key={id}>
+    <li key={id} className="d-flex justify-content-around">
+      <span>
+        {category}
+      </span>
       <span>{title}</span>
-      <span>{author}</span>
+      {/* <span>{author}</span> */}
```
Comment (mikenath223) [inline src/components/BookLists.js]: - [ ] Kindly remove this commented code.

File: src/components/BookLists.js [hunk 2]
```diff
@@ -27,7 +30,8 @@ const BookLists = (props) => {
 BookLists.propTypes = {
   title: PropTypes.string.isRequired,
   id: PropTypes.string.isRequired,
-  author: PropTypes.string.isRequired,
+  // author: PropTypes.string.isRequired,
```
Comment (mikenath223) [inline src/components/BookLists.js]: - [ ] Kindly remove this commented code.

**Comment by mikenath223** [comment]
- [ ] Please update the readme to match [this template](https://github.com/microverseinc/readme-template).

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @peterdgreat, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**